azure-devops/azext_devops/dev/team/_help.py

```diff
@@ -38,6 +38,35 @@ def load_team_help():
     short-summary: Manage security groups
     """
 
+    helps['devops security group create'] = """
+    type: command
+    short-summary: Create a new Azure DevOps group or materialize an existing AAD group.
+    long-summary:
+    examples:
+          - name: Create an Azure DevOps Group with name and description
+            text: |
+              az devops security group create --name 'Some group name' --description
+              'Something to describe this group' 
+              
+          - name: Materialize an existing AAD Group with its origin ID/AAD object ID
```

Comment:
IMO, materialize seems to be a confusing terminology or is it an understood terminology? Is there any documentation around the same that uses "materialize" as the terminology?

Reply:
No, there is no documentation on the terminology